src/meta-srv/src/table_meta_alloc.rs

```diff
@@ -66,7 +70,30 @@ impl TableMetadataAllocator for MetaSrvTableMetadataAllocator {
         )
         .await
         .map_err(BoxedError::new)
-        .context(meta_error::ExternalSnafu)
+        .context(meta_error::ExternalSnafu)?;
+
+        // Allocates a wal options for each region. The allocated wal options is encoded immediately.
+        let num_regions = region_routes.len();
+        let wal_options = self
+            .wal_options_allocator
+            .alloc_batch(num_regions)
+            .into_iter()
+            .map(|wal_options| {
+                serde_json::to_string(&wal_options)
+                    .context(EncodeWalOptionsToJsonSnafu { wal_options })
+            })
+            .collect::<MetaResult<Vec<_>>>()?;
+        let region_wal_options = region_routes
+            .iter()
+            .map(|route| route.region.id.region_number())
+            .zip(wal_options)
+            .collect();
+
```

Comment:
Add some logs for future debugging purposes.

Reply:
Added some logs. More logs will be added in the following PRs.